integrations/terraform/envs/platform/config.tf

```diff
@@ -0,0 +1,125 @@
+locals {
+  # This prefix will be used for all of your created resources
+  prefix = "platform"
+  # The AWS account ID you want to add your AWS resources to
+  aws_account_id = "012345678901"
+  # Name of the AWS account
+  account_name = "data-platform"
+  # Region where you want to deploy resources to
+  aws_region = "eu-west-1"
+
+  # Choose whether you want to set up Conveyor integrations
+  conveyor_enabled = false
+
+  # Tags that will be set for all AWS services
+  tags = {
+    Environment = local.account_name
+    Project     = local.prefix
+  }
+
+  # Choose whether you want to create a jump host
+  create_jump_host = true
+  # Choose whether you want to set-up a datahub data catalog on a EKS cluster for your data platform
+  create_datahub = false
+
+  # Account level configuration. This information is only relevant if you enable services running on the EKS cluster
+  account = {
```

Comment:
Probably requires a cleanup if we only keep the data_products module?

Reply:
It depends for me, do we want to offer this as an open source offering as well? If yes, do we want to include it in this repo or only provide the data_products module in this repo? Having it here might help in terms of integrating with AWS